test/unit/ui/app/actions.spec.js

```diff
@@ -786,9 +786,9 @@ describe('Actions', () => {
 
     it('calls signTypedMsg in background with no error', async () => {
       const store = mockStore()
-      signTypedMsgSpy = sinon.spy(background, 'signTypedMessage')
+      signTypedMsgSpy = sinon.stub(background, 'signTypedMessage')
 
-      await store.dispatch(actions.signTypedMsg(msgParamsV3))
+      store.dispatch(await actions.signTypedMsg(msgParamsV3))
```

Comment:
This seems odd. We should be able to `await` the dispatch of async actions 🤔

Reply:
Yeah, this is equivalent to removing `await` altogether. `await`-ing `signTypedMsg` doesn't do anything because it doesn't return a Promise - it returns a function (which itself returns a promise).

Reply:
I'd suggest doing something like `signTypedMsgStub.callsFake((_, callback) => callback())` to ensure the stub doesn't just hang.

Reply:
Ah, well, I guess just letting it hang works too 😅

Reply:
For the `callback`, doesn't `sinon.restore()` clean that up?

Reply:
What I mean is that the `signTypedMsg` isn't calling its callback, so that part of the action isn't executed. So in a sense the action only gets about halfway. I misspoke; it's not fair to describe that as "hung", it's just never called.